packages/react-error-overlay/middleware.js

```diff
@@ -11,6 +11,11 @@
 const { launchEditor } = require('react-dev-utils/launchEditor');
 
 module.exports = function createLaunchEditorMiddleware() {
+  if (process.platform === 'win32' && !process.env.REACT_EDITOR) {
+    const { launchPowerShellAgent } = require('react-dev-utils/launchEditor');
+    launchPowerShellAgent();
```

Comment:
Let's call it `tryLaunchPowerShellAgent` to make it clear it's always safe to call.